src/components/Hightlight/Example.md

```diff
@@ -0,0 +1,25 @@
+**Elementary**
```

Comment:
Could you also provide an example of how to change the color of a highlighted text?

Reply:
Working on it!